### ISSUE TITLE
Unexpected scale for pandas datetime column

### ISSUE THREAD

**Issue by harrylojames** [juba/pyobsplot#19]
datetime64[ns] objects aren't converted to javascript date objects. Presume this is intended as per the [section on datetimes](https://juba.github.io/pyobsplot/usage.html#datetime-objects). Would be great if it were possible! 

```
import pandas as pd
from pyobsplot import Plot

df = pd.DataFrame({'full_date_time': {0: pd.Timestamp('2022-12-01 00:00:00'),
  1: pd.Timestamp('2022-12-01 01:00:00')},
 'value': {0: 20.0, 1: 18.0}})

Plot.plot({"marks":[Plot.dot(df, {"x":"full_date_time", "y": "value"})]})
```

![image](https://github.com/juba/pyobsplot/assets/51630653/d9dfeecf-c1e0-4e71-8341-ee7d0db565e5)

**Comment by harrylojames** [comment]
Excuse my ignorance on what's been written under the hood but Quarto's [ojs_define ](https://github.com/quarto-dev/quarto-cli/blob/e09272956ab01841c85af64825c21bbf493cdfad/src/resources/jupyter/lang/python/setup.py#L68)seems to work pretty smoothly for converting panda's objects if there are any tricks not yet borrowed.

**Comment by juba** [comment]
Yes, you're right, date objects are converted correctly, but timestamps and datetime are not. 

I'll take a look, thanks for reporting !

**Comment by juba** [comment]
I think this should be fixed in the new 0.3.6 release, but let me know if it is working correctly for you.

Thanks !

**Comment by harrylojames** [comment]
2 for 2 🎉thank you!
